settings.gradle.kts

```diff
@@ -2,8 +2,29 @@ pluginManagement {
     repositories {
         maven(url = "http://dl.bintray.com/kotlin/kotlin-eap")
         mavenCentral()
+        jcenter()
+        google()
         maven(url = "https://plugins.gradle.org/m2/")
     }
+    resolutionStrategy {
```

Comment:
Do we really need this?

Reply:
as for new plugins block, Yup.

Reply:
if google and jetbrains cared about it more they may put gradle plugin markersin the repo.. but well not yet..